src/main/sensors/gyro.c

```diff
@@ -625,8 +626,12 @@ float dynThrottle(float throttle) {
 void dynLpfGyroUpdate(float throttle)
 {
     if (gyro.dynLpfFilter != DYN_LPF_NONE) {
-        const unsigned int cutoffFreq = fmax(dynThrottle(throttle) * gyro.dynLpfMax, gyro.dynLpfMin);
-
+        static unsigned int cutoffFreq;
```

Comment:
Why are you making this `static` here?

Reply:
You're right, no need to use `static` here. Fixed in pid.c too